wzdx/raw_to_standard/planned_events.py

```diff
@@ -202,6 +203,7 @@ def hex_to_binary(hex_string):
     "left lane": 'general',
     "center lane": 'general',
     "middle two lanes": 'general',
+    'general': 'general',
```

Comment:
```suggestion
    "general": 'general',
```
Just to keep consistent